test/new_relic/agent/pipe_channel_manager_test.rb

```diff
@@ -360,4 +364,12 @@ def reset_lifetime_counts!(container)
     buffer.instance_variable_set(:@captured_lifetime, 0)
     buffer.instance_variable_set(:@seen_lifetime, 0)
   end
+
+  def skip_for_ruby_2_5_9_and_rails_6_x
+    # TODO: MAJOR VERSION - remove this method and its calls when Ruby 2.5 is no longer supported
+    skip if RUBY_VERSION == '2.5.9' &&
+      defined?(Rails::VERSION::STRING) &&
+      # version string starts with 6.0 or 6.1
+      Rails::VERSION::STRING =~ /\A6\.(0|1)/
```

Comment:
Given that there won't be a Rails 6.2, I think checking for `Rails::VERSION::MAJOR` and then doing `Rails::VERSION::MAJOR == 6` would suffice. Given the context (test code only) there's no need to change it though.

Reply:
Thanks for this suggestion. I thought about that too, even though it seems unlikely/unprecedented 6.2 would ever come out, I figured it might be better to specify.